### PR TITLE
[exporter/signalfx]  Enable the exporting of seven Kubernetes metrics used in Splunk/SignalFx content by default

### PR DESCRIPTION
**Pull request by jvoravong**
**Description:**
This PR updates the default metrics that the SignalFx exporter excludes from exporting entirely. Seven Kubernetes metrics were being excluded that are used in Splunk content. With these changes, some Splunk content will start to populate properly.

Since these changes are only needed for Splunk Content, I thought updating the SignalFx exporter alone would be best. There is an alternative approach I will mention. We could update the default metrics that are enabled in the kubletstatsreceiver, k8sclusterreceiver, and k8seventsreceiver using [metadata.yaml](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/adffe2242ff1721928c79c5bc666b5fe0e06c76f/receiver/kubeletstatsreceiver/metadata.yaml#L58) files for filtering. Using the metadata.yaml file approach would increase the scope of stake holders and coordination, so I'd like to save this work for the future.

**Link to tracking Issue:** 
Did not create an opentelemetry-collector-contrib issue to track this change due to this being smaller issue with few stake holders.

**Testing:**
Updated unit tests. Tested Manually.

**Documentation:**
Only updated the CHANGELOG.md file. No other documentation updates are needed.

### CODE REVIEW

File: exporter/signalfxexporter/internal/translation/default_metrics.go [hunk 1]
```diff
@@ -118,10 +118,17 @@ exclude_metrics:
   - '!k8s.container.memory_limit'
   - '!k8s.container.cpu_limit'
 
+  # matches all container request metrics but k8s.container.cpu_request and k8s.container.memory_request
   - /^k8s\.container\..+_request$/
```
Comment (dmitryax) [inline exporter/signalfxexporter/internal/translation/default_metrics.go:122]: Should we just remove this line instead? Wouldn't it be the same effect?

File: exporter/signalfxexporter/internal/translation/default_metrics.go [hunk 2]
```diff
@@ -118,10 +118,17 @@ exclude_metrics:
...
+  - '!k8s.container.memory_request'
+  - '!k8s.container.cpu_request'
 
-  # matches any node condition but k8s.node.condition_ready
+  # matches any node condition but memory_pressure, network_unavailable, out_of_disk, p_i_d_pressure, and ready
   - /^k8s\.node\.condition_.+$/
```
Comment (dmitryax) [inline exporter/signalfxexporter/internal/translation/default_metrics.go:127]: Same q here

Comment (jvoravong) [reply exporter/signalfxexporter/internal/translation/default_metrics.go:127]: Kubernetes has the default node conditions which are well documented and are what we exclude here. Kubernetes built  a [node problem detector](https://github.com/kubernetes/node-problem-detector) you can install that reports several more node conditions. Splunk content has prebuilt filters for the default node conditions.

File: exporter/signalfxexporter/internal/translation/default_metrics.go (hunk 1 above)
Comment (dmitryax) [reply exporter/signalfxexporter/internal/translation/default_metrics.go:122]: Ok sounds good

**Review by dmitryax — APPROVED** [review]

**Pull request merged**